src/utils/Utils.ts

```diff
@@ -69,3 +69,11 @@ export function getFormattedOffset(utcOffset: number): string {
 
   return `${offsetOperator}${offsetHours}:${offsetMinutes}`;
 }
+
+/**
+ * Returns a Crypto object, either from the window or from the node:crypto package.
+ * @returns Get the crypto object from the global scope.
+ */
+export function getCrypto(): Crypto {
```

Comment:
When running unit tests, it's node env not browser env, we need this to be able to pass the unit test.
Check https://nodejs.org/api/crypto.html#determining-if-crypto-support-is-unavailable

Reply:
We do not support Node enviroment though. In our unit test can we catch the error and just return math.random instead?

Reply:
We did mock `crypto.getRandomValues()` here:
https://github.com/aws/amazon-chime-sdk-js/blob/629b521b94354165200cf03b1353cefcb247ebb4/test/dommock/DOMMockBuilder.ts#L1526

But the problem I'm having now is, when running unit test using node 20, it throws this error:
```
     TypeError: Cannot set property crypto of #<Object> which has only a getter
      at new DOMMockBuilder (test/dommock/DOMMockBuilder.ts:1525:21)
      at Context.<anonymous> (test/messagingsession/MessagingSessionConfiguration.test.ts:16:22)
      at processImmediate (node:internal/timers:478:21)
```

I'm trying to resolve this and then we should be good.

Reply:
Hmm so maybe we can just check on whether window.crypto exists, log a warning and fall back to Math.random instead? We do not support Node so kinda prefer just fallback to something else.

Reply:
Updated!